satpy/writers/__init__.py

```diff
@@ -293,20 +290,15 @@ def __init__(self, *config_files, **kwargs):
         self.add_config_to_tree(*config_files)
 
     def add_config_to_tree(self, *config_files):
-        conf = configparser.ConfigParser(allow_no_value=True)
-        for fn in config_files:
-            if isinstance(fn, str):
-                conf.read(fn)
-            else:
-                conf.readfp(fn)
+        conf = {}
+        for config_file in config_files:
+            with open(config_file) as fd:
+                conf = recursive_dict_update(conf, yaml.load(fd))
```

Comment:
My preference would be that we do a shallower dict update. At least that's my first instinct. You don't want someone who completely changed a certain enhancement entry or created what they thought was a new one that happened to have the same name as an existing builtin to only have the operations updated or slightly changed.